cg/meta/upload/scout/scout_config_builder.py

```diff
@@ -41,6 +41,7 @@ def add_common_info_to_load_config(
         load_config.family = analysis.case.internal_id
         load_config.family_name = analysis.case.name
         load_config.owner = analysis.case.customer.internal_id
+        load_config.status = "prioritized" if analysis.case.is_prioritized else None
```

Comment:
Should this logic be in a separate funtion?

Reply:
I would add the a constant called SCOUT_PRIORITIZED_STATUS in `cg/constants/subject.py`

Reply:
Finally a situation where we can use IntEnum as intended 🥳 If this looks messy to you can move it to a method
```suggestion
        load_config.status = SCOUT_PRIORITIZED_STATUS if analysis.case.priority >= Priority.priority else None
```

Reply:
Ahhh! Great suggestions! Will change!